eap-xp/image.yaml

```diff
@@ -0,0 +1,134 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp1-openjdk11-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform XP 1.0 OpenShift container image."
+version: "1.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-7-xp1-openjdk11-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP XP 1.0"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP XP 1.0"
```

Comment:
I'm not sure if we should put the version in that field?

Reply:
+1, we shouldn't

Reply:
Regarding 1.0.0, it doesn't seem needed to me, unlike say :7.3.1 in the EAP case. But we can add it if we think it might be useful, and just have it reference :1.0